R/postprocess.R

```diff
@@ -1,3 +1,137 @@
+#' Extract samples of the delay distribution parameters
+#'
+#' @param fit A model fit with `epidist::epidist`
+#' @inheritParams brms::prepare_predictions
+#' @param ... Additional arguments passed to `brms::prepare_predictions`
+#' @family postprocess
+#' @autoglobal
+#' @export
+predict_delay_parameters <- function(fit, newdata = NULL, ...) {
+  args <- list(...)
+  if ("newdata" %in% names(args)) {
```

Comment:
as new data is an arg we don't need an if check here and it won't be activated regardless. We need to check based on if its NULL I believe

Reply:
Fixed